R/check_tr_trdtc_across_visit.R

```diff
@@ -4,6 +4,8 @@
 #'  across multiple visits when TRTESTCD is "LDIAM". Only applies to assessments by investigator.
```

Comment:
@sarabodach same question here - remove TR records with missing date before running check?

Reply:
I wonder if a separate check looking for TESTCDs with any date mapped.  And then of those looking for missing dates makes sense.